### DISCUSSION TITLE
How to reset config to defaults?

### DISCUSSION THREAD

**Discussion by veggie2u** [RaspAP/raspap-webgui#1293] [Q&A]
I had the pi wifi as AP, a usb wifi as client on home network, and was trying to make eth0 like the AP too. I must have messed up something as my AP now says down (oled screen). I am able to ssh to the pi from another machine, but can't seem to reset the config. I did a install with --update. No effect. Then I did an install without that flag. Still no effect. I was hoping it would reset the config and I could start over, but the AP still says down. 

Anyone know how to reset the config to defaults? I am hoping not to start from a fresh Raspberry OS install.

**Comment by billz** [comment]
See [restoring your default settings](https://docs.raspap.com/defaults/#restoring-settings). You may also restore them [manually](https://docs.raspap.com/manual/#default-configuration).

If you used the `--upgrade` switch (there is no --update) this upgrades your install to the latest release _without_ modifying your configuration.
